turbine/src/sigverify_shreds.rs

```diff
@@ -145,12 +145,22 @@ fn run_shred_sigverify<const K: usize>(
     stats.num_batches += packets.len();
     stats.num_packets += packets.iter().map(PacketBatch::len).sum::<usize>();
     stats.num_discards_pre += count_discards(&packets);
+    // Repair shreds include a randomly generated u32 nonce, so it does not
+    // make sense to deduplicate them (i.e. they are not duplicate of any other
+    // shred).
+    // If the nonce is excluded from the deduper then false positives might
+    // prevent us from repairing a block until the deduper is reset after
+    // DEDUPER_RESET_CYCLE. A workaround is to also repair "coding" shreds to
```

Comment:
Remind me, we pick numbers such that a false positive should be quite rare with current load right ?

Reply:
That is set below at 0.1%
https://github.com/anza-xyz/agave/blob/79692775a/turbine/src/sigverify_shreds.rs#L41